client/chromez-behaviors.js

```diff
@@ -28,17 +29,17 @@ ChromeZBehaviors.AJAXBehavior = {
     }
   },
   handleResponse: function(data) {
-    var query = this.searchQueries[data.model.index];
+    var searchQuery = this.searchQueries[data.model.index];
```

Comment:
This rename is *super marginal*:
(1) you almost don't make use of the parameter at all, so you're adding a lot of noise to the review
(2) there's absolutely nothing wrong with the old name
(3) you don't need to do the rename for your code to work

I'd avoid things like this in the future.